src/HypothesisTests.jl

```diff
@@ -111,22 +111,25 @@ function Base.show(io::IO, test::T) where T<:HypothesisTest
     println(io, "    value under h_0:         $param_under_h0")
     println(io, "    point estimate:          $param_estimate")
     if has_ci
-        println(io, "    95% confidence interval: $(StatsBase.confint(test))")
+        ci = map(x -> round.(x, 4), StatsBase.confint(test))
```

Comment:
No need for the dot, right? Also no need for a line break below.

Reply:
Thanks. But it doesn't work without the dot
`MethodError: no method matching round(::Tuple{Float64,Float64}, ::Int64)`

Reply:
Sorry, I hadn't realized there could be multiple CIs.

Something I noticed in the tested outputs is that `0.3` is not printed as `0.300`, which would be better to make it clear that there are three digits of precision and to ease the comparison with other values. You can achieve that with `@sprintf("%.3f", x)`.

Reply:
Unfortunately, that doesn't seem to work on the array of tuples:
```
using HypothesisTests
d = [[762,484] [327,239] [468,477]]
m = PowerDivergenceTest(d)
ci = map(x -> round.(x, 4), StatsBase.confint(m))
@sprintf("%.4f", ci)

MethodError: no method matching isfinite(::Array{Tuple{Float64,Float64},1})
```

But I probably don't understand what you have in mind.

Reply:
Yes, you'll have to use something like `map(x -> @sprintf("%.4f", x), ci)` just like you did for `round`. BTW, no need to call `round` beforehand, so you can just replace that call.

Reply:
Yeah, I tried that yesterday but unfortunately that still doesn't work with the array of tuples.
```
using HypothesisTests
d = [[762,484] [327,239] [468,477]]
m = PowerDivergenceTest(d)
ci = StatsBase.confint(m)
map(x -> @sprintf("%.4f", x), ci)

MethodError: no method matching isfinite(::Tuple{Float64,Float64})
```

Do I need to loop through the array?

Reply:
Ah, yes, you'll need two nested calls to `map`.

Reply:
It also works with a comprehension
```
ci = [ map(x -> @sprintf("%.4f", x), y) for y in StatsBase.confint(m) ]
```
Unfortunately, ``println()`` then produces
```
95% confidence interval: Tuple{String,String}[("0.2579", "0.2955"), ("0.1571", "0.1947"), ("0.1001", "0.1378"), ("0.0682", "0.1058"), ("0.1513", "0.1889"), ("0.1545", "0.1922")]
```
So it's now an array of tuples of strings (albeit rounded nicely). Would you prefer this or should I go back to the just rounded output for now? Following your suggestion below, I think it makes sense to restructure this in a table anyways.

Reply:
Ah, right, you'd need to print the parts of the tuple manually. Let's keep `round` for now and improve this later by using a table instead.

Reply:
Perfect. Thanks for your help here!